src/SmartComponents/redux/reducers.js

```diff
@@ -18,3 +18,33 @@ export function composeReducer(state = { }, action) {
             return state;
     }
 }
+
+const initialPendingComposeState = {
+    release: {
+        arch: 'x86_64',
+        release: 'rhel-8',
+    },
+    uploadDestinations: {
+        aws: false,
+        azure: false,
+        google: false,
+    },
+};
+
+export function pendingCompose(state = initialPendingComposeState, action) {
```

Comment:
I think it would be good to split the reducers into different files. So `/reducers/composes` and `/reducers/pendingCompose`